trlc/trlc.py

```diff
@@ -886,12 +886,11 @@ def get_status(parser):
                        parser.cu.package.name))
 
     if ok:
-        if options.error_on_warnings and mh.warnings:  # pragma: no cover
+        if options.error_on_warnings and mh.warnings \
```

Comment:
Can you please add brackets for better clarity?
`(options.error_on_warnings and mh.warnings) or mh.errors`